src/rez/cli/_entry_points.py

```diff
@@ -0,0 +1,215 @@
+"""
+Entry points.
+"""
+
+### Utility functions
+
+def get_specifications():
+    """Get entry point specifications
+
+    See:
+    * https://pythonhosted.org/distlib/reference.html#distlib.scripts.ScriptMaker.make_multiple
+    * https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation
+
+    Example return value:
+
+        {
+            "rez-env": "rez-env = rez.cli._entry_points.run_rez_env",
+            ...
+        }
+
+    Returns:
+        dict (str, str): The specification string for each script name.
+    """
+    import sys
+
+    specs = {}
+
+    for attr, obj in sys.modules[__name__].__dict__.iteritems():
+        scriptname = getattr(obj, "__scriptname__", None)
+        if scriptname:
+            spec = "%s = rez.cli._entry_points:%s" % (scriptname, attr)
+            specs[scriptname] = spec
+
+    return specs
+
+
+def scriptname(name):
+    def decorator(fn):
+        setattr(fn, "__scriptname__", name)
+        return fn
+    return decorator
+
+
+### Entry points
+
+@scriptname("rez")
+def run_rez():
+    from rez.cli._main import run
+    return run()
+
+
+@scriptname("rezolve")
+def run_rezolve():
+    # alias for osx, where rez is a different tool
+    # https://www.unix.com/man-page/osx/1/REZ/
+    from rez.cli._main import run
+    return run()
+
+
+@scriptname("bez")
+def run_bez():
+    # TODO: Deprecate. Use custom build commands instead.
+    # https://github.com/nerdvegas/rez/wiki/Building-Packages#custom-build-commands
+    from rez.cli._bez import run
+    run()
+
+
+@scriptname("_rez-complete")
+def run_rez_complete():
+    from rez.cli._main import run
+    return run("complete")
+
+
+@scriptname("_rez-fwd")
+def run_rez_fwd():
+    from rez.cli._main import run
+    return run("forward")
+
+
+@scriptname("rez-bind")
+def run_rez_bind():
+    from rez.cli._main import run
+    return run("bind")
+
+
+@scriptname("rez-build")
+def run_rez_bind():
+    from rez.cli._main import run
+    return run("bind")
```

Comment:
I think you copy/paste a little too fast :stuck_out_tongue: It should be `build` and the function name should also contain `build` as right now it override the bind. Same for `rez-config`.

Reply:
thanks for the catch, fixes coming